zerver/lib/markdown/__init__.py

```diff
@@ -2423,6 +2478,7 @@ def build_inlinepatterns(self) -> markdown.util.Registry[markdown.inlinepatterns
             UserGroupMentionPattern(mention.USER_GROUP_MENTIONS_RE, self), "usergroupmention", 65
         )
         reg.register(LinkInlineProcessor(markdown.inlinepatterns.LINK_RE, self), "link", 60)
+        reg.register(ImageInlineProcessor(markdown.inlinepatterns.IMAGE_LINK_RE, self), "image", 58)
```

Comment:
How did you pick 58 for this? Does it align with the ordering in python-markdown upstream?

Reply:
I do! This new processor needed to be ordered such that it ran before LinkInlineProcessor and after AutoLink, so I picked a random number between their priorities instead of tweaking all to maintain the multiples of 5, which I never got around to testing